### DISCUSSION TITLE
What are your BoTorch pain points? What should run faster or less memory?

### DISCUSSION THREAD

**Discussion by esantorella** [meta-pytorch/botorch#1369] [Q&A]
We are looking for feedback on areas for improvement. While I'm initially focusing on speed and memory issues, we'd also love to hear about issues of API ergonomics and documentation, or any other feedback you have. So I'm asking anyone interested:

What combinations of models and acquisition functions do you use most often? Which cause a lot of pain in terms of speed, memory, or ease of use? Is there anywhere you suspect there's low-hanging fruit for performance improvements?

Minimal working examples are ideal, but a verbal description of your pain points is also helpful. Thanks in advance!

**Comment by saitcakmak** [comment]
cc @raulastudillo06 for issues you or others from the group ran into

**Reply by Balandat** [reply]
We're running in an issue with pyro NUTS on GPU when using the torch jit: https://github.com/pyro-ppl/pyro/issues/3136#issue-1364061798. It would be good to understand how much of a speedup we're getting from using the jit here both on CPU and GPU.
